tests/src/test/scala/org/http4s/multipart/MultipartSpec.scala

```diff
@@ -123,25 +118,25 @@ Content-Type: application/pdf
 
 ------WebKitFormBoundarycaZFo8IAKVROTEeD--
       """.replaceAllLiterally("\n", "\r\n")
-    val header = Headers(
-      `Content-Type`(
-        MediaType.multipartType("form-data", Some("----WebKitFormBoundarycaZFo8IAKVROTEeD"))))
-    val request = Request[IO](
-      method = Method.POST,
-      uri = url,
-      body = Stream.emit(body).through(text.utf8Encode),
-      headers = header)
-
-    val decoded = EntityDecoder[IO, Multipart[IO]].decode(request, true)
-    val result = decoded.value.unsafeRunSync()
-
-    result must beRight
-  }
+        val header = Headers(
+          `Content-Type`(MediaType.multipart.`form-data`
+            .withExtensions(Map("boundary" -> "----WebKitFormBoundarycaZFo8IAKVROTEeD"))))
```

Comment:
This merge was so bad I started over from what @jmcardon had, and then I summoned @cquiroz to discuss what happened to the old `multipart` convenience constructor, and now I see that it exists.  I guess I should change thes eto look like the old?

Reply:
If possible yes plz